Automattic/jetpack
# Social Notes: Hide custom message social note classic

This small change hides the classic editor's custom message box for social notes.
The same change for Gutenberg is created in #35536

## Proposed changes:

* Put the rendering of the custom message input behind a decision check

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/198
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Check out this PR. If you want to be able to share with working permalinks, put #35462 under it.
* Activate the Classic editor plugin.
* Create a new social note. You should see that after hitting edit in the sidebar settings where you can toggle the settings, the custom message box does not appear.
* Do the same with a normal post, the custom message box should be there.

Also, check out the sharing functionality. If you checked this PR onto the one mentioned above, it should work fine

![CleanShot 2024-02-08 at 14 39 23 png](https://github.com/Automattic/jetpack/assets/36671565/1b5a8268-d092-4b99-adf9-548629f35906)
